### PR TITLE
Adding Balena as a method to try WPE

### PR DESCRIPTION
**Pull request by manuelafm**
Broken link to last image to be addressed at the source, as it contains a   typo that can be fixed without breaking anything else (one hopes...): https://github.com/Igalia/balena-wpe/pull/16

----

Site preview: https://igalia.github.io/wpewebkit.org/mafm-exploring/

### CODE REVIEW

File: about/balena-wpe.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+## Demos
+
+| Name | Animation |
+|:----|:---:|
+| CSS transformation and animations in [WebKit poster circle example](https://webkit.org/blog-files/3d-transforms/poster-circle.html) | ![Image: WebKit poster circle demo](https://raw.githubusercontent.com/igalia/balena-wpe/master/static/resources/postercircle.gif) |
+| Playing an embedded video from [wpewebkit.org](https://wpewebkit.org/) | ![Image: Embedded video demo](https://raw.githubusercontent.com/igalia/balena-wpe/master/static/resources/wpewebkitorg.gif) |
+| Playing [BouncyBalls](https://bouncyballs.org/) | ![Image: Playing BouncyBalls demo](https://raw.githubusercontent.com/igalia/balena-wpe/master/static/resources/bouncyballs.gif) |
+| Rendering WebGL [Aquarium](https://webglsamples.org/aquarium/aquarium.html) demo | ![Image: Rendering WebGL Aquarium demo](https://raw.githubusercontent.com/igalia/balena-wpe/master/static/resources/webgl_aquarium.gif) |
+| Rendering WebGL [Abstract Shapes](https://mrdoob.neocities.org/023/) | ![Image: Rendering WebGL Abstract Shapes demo](https://raw.githubusercontent.com/igalia/balena-wpe/master/static/resources/webgl_abstractshapes.gif) |
```
Comment (aperezdc) [inline about/balena-wpe.md:128]: This URL seems to be wrong, the server returns 404 not found:

- https://raw.githubusercontent.com/igalia/balena-wpe/master/static/resources/webgl_abstractshapes.gif

**Review by aperezdc — CHANGES_REQUESTED** [review]
One of the images seems to be missing from the server, please take a look.

Other than the missing image, I think the content is good. The text could probably use another proofreading by @bkardell and/or @meyerweb in case I missed some typo of grammar issue.

File: about/balena-wpe.md (hunk 1 above)
Comment (manuelafm) [reply about/balena-wpe.md:128]: That would be fixed with https://github.com/Igalia/balena-wpe/pull/16
@psaavedra , can we go ahead and merge that change?

Comment (aperezdc) [reply about/balena-wpe.md:128]: Wouldn't it be better to host copies of the images at `wpewebkit.org`?

Comment (manuelafm) [reply about/balena-wpe.md:128]: I don't have a strong opinion, so I'll include images then.

Comment (manuelafm) [reply about/balena-wpe.md]: The images can not be seen with the given syntax in Markdown in the preview site.  Should I do it with relative paths?  I think that it's better to keep it in Markdown though, and not do it with html `` tag as in `about/architecture.md`

Comment (aperezdc) [reply about/balena-wpe.md]: The images show just fine here in the preview. I suppose the GitHub pages service took a little while to properly update the content 😉

**Review by aperezdc — APPROVED** [review]
I'll wait a day or two before merging this in case someone else wants to proofread, but all looks good to me now. Thanks for the new content!

**Pull request merged**